### PR TITLE
wait for PublisherMaster to be idle before killing. Fix #6664

### LINKED ISSUE #6664: make Publisher stop.sh "safe"
it should check that PublisherMaster is in the wait time between cycles, wait if not, and onlly stop when it is Idle.
Currentlly Publisher duty cycle is low, so this will be OK.
Will only worry when needed about something a' la TW where a STOP message is put in the PublisherMaster queue and acted upon automatically.

### PR DESCRIPTION
**Pull request by belforte**

### CODE REVIEW

**Comment by belforte** [comment]
I have tested this, but since killing processes is always a dangerous operation, please have a look and test it yourself as well before putting in production

File: src/script/Deployment/Publisher/stop.sh [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+  sleep 10
+  echo -n "."
+  if (( nIter%6  == 0 )); then
+    minutes=$((nIter/6))
+    echo -n "${minutes}m"
+  fi
+done
+echo ""
+echo "Publisher is in waiting now. Killing PublisherMaster"
+pkill -f PublishMaster
```
Comment (ddaina) [inline src/script/Deployment/Publisher/stop.sh:47]: I think it should be `PublisherMaster` instead of `PublishMaster`, no?

Comment (belforte) [reply src/script/Deployment/Publisher/stop.sh]: thanks. I guess it was right, then I removed to test w/o disrupting, then failed when typing it back, fixes in new commit

**Review by ddaina — APPROVED** [review]
tested on preprod, lgtm

**Pull request merged**